common/scoring.ts

```diff
@@ -62,7 +63,18 @@ export function scoreUsersByContract(contract: Contract, bets: Bet[]) {
       return { userId, payout }
     })
 
-  const netPayouts = [...resolvePayouts, ...salePayouts, ...investments]
+  const investmentValues = openBets.map((bet) => {
+    const { userId } = bet
+    const investmentValue = computeInvestmentValueForBet(bet, contract)
+    return { userId, payout: investmentValue }
+  })
+
+  const netPayouts = [
+    ...resolvePayouts,
+    ...salePayouts,
+    ...investments,
+    ...investmentValues,
```

Comment:
As far as I understand it, this is what we want to add to the current leaderboard. 

Before this PR:
The group leaderboard score for each user would only be based on profits on resolved markets. 
After this PR:
InvestmentValues is added, i.e the current value of the shares each user has on each **unresolved** contract in the group.

Reply:
I'll have more thoughts after all my meetings, but very quickly:

Investment value is not exactly right. If someone bets M$1000 that shouldn't automatically put them on the leaderboard. What we care about is current profit on their investment.

Profit is the current investment value minus the initial investment. You can also check how we calculate profit in `calculate.ts`.

Reply:
@jahooma Right, yeah makes sense. Will update after the standup.